src/libumf.c

```diff
@@ -63,12 +63,16 @@ void umfTearDown(void) {
 #endif
         umfIpcCacheGlobalTearDown();
 
+        fini_ze_global_state();
+        fini_cu_global_state();
+        LOG_DEBUG("UMF library finalized");
+
```

Comment:
Why do we need it earlier?

Reply:
Reverted, I need that to be able to free memory in fini_ze_global_state, but we don't actually need to keep the driver handles so it's not necessary.